applications/launchpad/gui-react/src/containers/SettingsContainer/DockerSettings/DockerImagesList/index.tsx

```diff
@@ -0,0 +1,75 @@
+import { useEffect } from 'react'
+import { useTheme } from 'styled-components'
+
+import { useAppSelector, useAppDispatch } from '../../../../store/hooks'
+import { actions } from '../../../../store/dockerImages'
+import {
+  selectDockerImages,
+  selectDockerImagesLoading,
+} from '../../../../store/dockerImages/selectors'
+import Text from '../../../../components/Text'
+import Loading from '../../../../components/Loading'
+import LoadingOverlay from '../../../../components/LoadingOverlay'
+import Tag from '../../../../components/Tag'
+import Button from '../../../../components/Button'
+import CheckIcon from '../../../../styles/Icons/CheckRound'
+import QuestionMarkIcon from '../../../../styles/Icons/Info1'
+import t from '../../../../locales'
+
+import { DockerRow, DockerList, DockerStatusWrapper } from './styles'
+
+const DockerImagesList = () => {
+  const theme = useTheme()
+  const dispatch = useAppDispatch()
+  useEffect(() => {
+    dispatch(actions.getDockerImageList())
+  }, [dispatch])
+
+  const dockerImages = useAppSelector(selectDockerImages)
+  const dockerImagesLoading = useAppSelector(selectDockerImagesLoading)
+
+  return (
+    <DockerList>
+      {dockerImagesLoading && <LoadingOverlay />}
+      {dockerImages.map(dockerImage => (
+        <DockerRow key={dockerImage.dockerImage}>
+          <Text style={{ flexBasis: '40%' }}>{dockerImage.displayName}</Text>
```

Comment:
Here is the issue on macOS:

![image](https://user-images.githubusercontent.com/11715931/175332450-a734fed4-66b8-49a8-b4dc-838829d9179b.png)

...but if we change the `flexBasis` to `38%`, it looks good:

![image](https://user-images.githubusercontent.com/11715931/175332627-245ad9e8-fb42-496d-a69d-b87657a5fe3f.png)

I can handle this later and check also how it looks on the Windows.